examples/workflows/plot_simulate_beta.py

```diff
@@ -0,0 +1,234 @@
+# contains information on the biophysics and geometry of each cell.
+net_jones = jones_2009_model()
+
+jones_rise = net_jones.cell_types['L5_pyramidal'].p_syn['gabab']['tau1']
+law_rise = net.cell_types['L5_pyramidal'].p_syn['gabab']['tau1']
+print(f'GABAb Rise (ms): {jones_rise} -> {law_rise}')
+
+jones_fall = net_jones.cell_types['L5_pyramidal'].p_syn['gabab']['tau2']
+law_fall = net.cell_types['L5_pyramidal'].p_syn['gabab']['tau2']
+print(f'GABAb Fall (ms): {jones_fall} -> {law_fall}\n')
+
+print('Apical Dendrite Channels:')
+print(net.cell_types['L5_pyramidal'].p_secs['apical_1']['mechs'].keys())
+print("\nBasal Dendrite Channels ('ca' missing):")
+print(net.cell_types['L5_pyramidal'].p_secs['basal_1']['mechs'].keys())
+
+###############################################################################
+# A major change to the Jones 2009 model is the addition of a
+# Martinotti-like recurrent tuft connection [3]_. This new connection
+# originates from L5 basket cells, and provides GABAa inhibition on
+# the distal dendrites of L5 basket cells.
+print('Recurrent Tuft Connection')
+print(net.connectivity[16])
+
+###############################################################################
+# The remaining changes to the connectivity was the removal of an
+# L2_basket -> L5_pyramidal GABAa connection, and replacing it with GABAb.
+print('New GABAb connection')
+print(net.connectivity[15])
+
+print('\nConnection Removed from Law Model')
+print(net_jones.connectivity[10])
+
+###############################################################################
+# To demonstrate sensory depression, we will add the drives necessary to
+# generate and ERP similar to
+# :ref:`evoked example <sphx_glr_auto_examples_plot_simulate_evoked.py>`,
+# but modified to reflect the parameters used in Law et al. 2021.
+# Specifically, we are considering the case where a tactile stimulus is
+# delivered at 150 ms. 25 ms later, the first input to sensory cortex arrives
+# as at the proximal drive to the cortical column. Proximal drive corresponds
+# projects from the direct thalamic nuclei. This is followed by one distal
+# representing projections from indirect thalamic nuclei, and a final late
+# proximal drive. It is important to note that the parameter values for each
+# are different from previous examples of the evoekd response. This reflects
+# the altered network dynamics due to the changes described above.
+def add_erp_drives(net, stimulus_start):
+    # Distal evoked drive
+    weights_ampa_d1 = {'L2_basket': 0.0005, 'L2_pyramidal': 0.004,
+                       'L5_pyramidal': 0.0005}
+    weights_nmda_d1 = {'L2_basket': 0.0005, 'L2_pyramidal': 0.004,
+                       'L5_pyramidal': 0.0005}
+    syn_delays_d1 = {'L2_basket': 0.1, 'L2_pyramidal': 0.1,
+                     'L5_pyramidal': 0.1}
+    net.add_evoked_drive(
+        'evdist1', mu=70.0 + stimulus_start, sigma=0.0, numspikes=1,
```

Comment:
this is just for my knowledge. Why is `numspikes=1` ?

Reply:
This is how it was simulated in the current study and the original 2009 paper https://journals.physiology.org/doi/full/10.1152/jn.00535.2009
> EVOKED RESPONSE INPUTS.
These were simulated as in Jones et al. (2007), with a sequence of FF input, followed by FB, followed by a reemergent late feedforward (LFF) input (schematically drawn in Fig. 9A). The timing of the input sequence was fixed as in Table 2. Each driving spike train consisted of a single presynaptic spike on each trial and the weights were distributed uniformly across the SI networks as in Table 2.

Reply:
As for theoretically why it's a single spike, it's a good question. Presumably there is some stochasticity in arrival time at the cortex, and `plot_simulate_evoked.py` is likely the more realistic simulation of an evoked input.

Reply:
I see ... I have to say I am super confused what's the interaction between `sigma=0`, `sync_within_trials=True`, `numspikes=1` and how it affects the histogram plot. I'll raise a separate issue though